gui-apps/darkman/darkman-2.2.0.ebuild

```diff
@@ -48,3 +46,12 @@ src_install() {
 	dodir /etc/env.d/
 	echo "XDG_DATA_DIRS=\"~/.local/share/\"" >> "${ED}"/etc/env.d/99darkman || die
```

Comment:
This is incorrect as it makes the default locations `XDG_DATA_DIRS="/usr/local/share:/usr/share"` unavailable, thus potentially breaking many applications.
Darkman also looks in `${XDG_DATA_HOME}` so this is redundant anyways.
https://gitlab.com/WhyNotHugo/darkman/-/blob/v2.2.0/scripts.go?ref_type=tags#L32

Reply:
Ok, I took it from previous ebuild without thinking too much, because I know little about /etc/env.d.